lms/djangoapps/labster_vouchers/views.py

```diff
@@ -83,15 +82,16 @@ def activate_voucher(request):
         ))
         return redirect(enter_voucher_url)
 
-    course_license = CourseLicense.objects.filter(license_code=license_code).first()
+    course_licenses = CourseLicense.objects.filter(license_code=license_code)
```

Comment:
What's the difference? Looks like the older version requires less SQL queries i.e. it's more efficient